chipsec/logger.py

```diff
@@ -27,8 +27,9 @@
 import binascii
 import sys
 import os
-from time import localtime, strftime
 import atexit
+from time import localtime, strftime
+from typing import Union, List, Tuple
```

Comment:
You're not using type hinting anymore. Recommend either adding type hinting or removing this import as it's not being used ATM.